control-plane/grpc/proto/v1/replica/replica.proto

```diff
@@ -157,6 +157,19 @@ message UnshareReplicaRequest {
   optional google.protobuf.StringValue pool_uuid = 5;
 }
 
+// Resize a replica to the requested size
+// For internal usage mostly.
+message ResizeReplicaRequest {
+  // id of the io-engine instance
+  string node_id = 1;
+  // uuid of the replica
+  google.protobuf.StringValue replica_id = 2;
```

Comment:
```suggestion
  string replica_id = 2;
```

Reply:
@tiagolobocastro All the messages in this file have replica_id as StringValue. Any particular reason to make that `string` here?

Reply:
they were added mistakenly... we've since switched to string

Reply:
ok, will change. I guess having it `string` will help simplify code as we may not need `Validated*` type in between.